diff --git a/node_modules/@astrojs/starlight/utils/createTranslationSystem.ts b/node_modules/@astrojs/starlight/utils/createTranslationSystem.ts
index 9ecf0dd..a51cbf8 100644
--- a/node_modules/@astrojs/starlight/utils/createTranslationSystem.ts
+++ b/node_modules/@astrojs/starlight/utils/createTranslationSystem.ts
@@ -128,6 +128,7 @@ function buildResources<T extends i18nSchemaOutput>(
 // strings. We let TypeScript merge them into a single union type so that plugins with a TypeScript
 // configuration preventing `UserI18nKeys` to be properly inferred can still get auto-completion
 // for built-in UI strings.
+// @ts-expect-error StarlightApp missing
 export type I18nKeys = keyof BuiltInStrings | UserI18nKeys | keyof StarlightApp.I18n;
 
 export type I18nT = TFunction<'starlight', undefined> & {
diff --git a/node_modules/@astrojs/starlight/utils/translations.ts b/node_modules/@astrojs/starlight/utils/translations.ts
index d4ffaca..e6d19fd 100644
--- a/node_modules/@astrojs/starlight/utils/translations.ts
+++ b/node_modules/@astrojs/starlight/utils/translations.ts
@@ -1,6 +1,9 @@
 import { getCollection, type CollectionEntry, type DataCollectionKey } from 'astro:content';
+// @ts-expect-error virtual module import
 import config from 'virtual:starlight/user-config';
+// @ts-expect-error virtual module import
 import project from 'virtual:starlight/project-context';
+// @ts-expect-error virtual module import
 import pluginTranslations from 'virtual:starlight/plugin-translations';
 import type { i18nSchemaOutput } from '../schemas/i18n';
 import { createTranslationSystem } from './createTranslationSystem';
